### PR TITLE
feat: sort holders by the most owned NFTs in activity

### LINKED ISSUE #5459: Sort holders by most owned NFTs in activity
I would like to see holders who has most NFTs on top 

Test
- https://beta.kodadot.xyz/rmrk2/collection/06ea5e9291a7e86837-CLOWN/activity
- 
https://user-images.githubusercontent.com/5887929/229174678-2cec3261-231f-4b1a-9244-016476606229.mov

### PR DESCRIPTION
**Pull request by leo-anderson-x**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5459
- [ ] Requires deployment <>

https://user-images.githubusercontent.com/128157824/229352655-63ff631c-755d-4716-aa9f-a01cd7a2d1d5.mp4

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

File: components/collection/activity/ownerInsightsTabs/HolderTab.vue [hunk 1]
```diff
@@ -71,7 +71,12 @@ const toggleNFTDetails = (flipperId: string) => {
   }
 }
 
-const holders = computed(() => Object.entries(props.owners || {}))
+const holders = computed(() =>
+  Object.entries(props.owners || {}).sort(
+    // sort by nft count: highest first
+    (a, b) => b[1].nftCount - a[1].nftCount
+  )
+)
```
Comment (vikiival) [inline components/collection/activity/ownerInsightsTabs/HolderTab.vue:79]: I do not honestly know what is data flow here but would be far better to do this on backend

**Review by vikiival — APPROVED** [review]
:adhesive_bandage:
Simple fix for a non-critical issue.

__https://gitmoji.dev__

**Comment by vikiival** [comment]
Cc @leo-anderson-x 

Screenshot 2023-04-02 at 22 35 26

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 32.56 USD/KSM ~ 0.307 $KSM
🧗 [DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C](https://kusama.subscan.io/account/DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C)
🔗 [0x3522d50c156f74af8a25ff6e2f84994a15fbb6be6303b7a4539af8724768baca](https://kusama.subscan.io/extrinsic/0x3522d50c156f74af8a25ff6e2f84994a15fbb6be6303b7a4539af8724768baca)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**